guides/v2.2/frontend-dev-guide/themes/js-bundling.md

```diff
@@ -1 +1,135 @@
-../../../v2.1/frontend-dev-guide/themes/js-bundling.md
\ No newline at end of file
+---
+group: fedg
+subgroup: A_Themes
+title: JavaScript Bundling
+menu_title: JavaScript Bundling
+menu_order: 9000
+version: 2.2
+github_link: frontend-dev-guide/themes/js-bundling.md
+functional_areas:
+  - Frontend
+  - Theme
+---
+
+JavaScript bundling is an optimization technique you can use to reduce the number of server requests for JavaScript files.
+Bundling accomplishes this by merging multiple JavaScript files together into one file to reduce the number of page requests.
+
+## Enable JavaScript bundling
+
+Magento must be set to [production mode][production-mode] in order for bundling to work.
+
+To activate JavaScript bundling:
+
+- Enter the following command to make
+    ```bash
+    php -f bin/magento config:set dev/js/enable_js_bundling 1
+    ```
+    
+<div class="bs-callout bs-callout-info" markdown="1">
+Enable Javascript Bundling configuration is not longer available in Magento version 2.2 and next version. 
+</div>    
+
+To optimize bundling, set the following settings to `Yes`:
+
+* `Javascript Settings -> Merge JavaScript Files`
+* `Javascript Settings -> Minify Javascript Files`
+* `Static Files Settings -> Sign Static Files`
```

Comment:
Sign Static Files doesn't optimize bundling, but does allow for cache busting. Perhaps that would be beneficial to clarify?